dot/core/service.go

```diff
@@ -282,11 +280,14 @@ func (s *Service) handleBlocksAsync() {
 				continue
 			}
 
+			prev := s.blockState.BestBlockHash()
```

Comment:
nit: would suggest renaming from `prev` to `bestBlockHash` since `prev` is not a clear name